Eatery Blue/UI/HomeScreen/HomeModelController.swift

```diff
@@ -206,7 +206,21 @@ class HomeModelController: HomeViewController {
             }
         }
         
-        let openEateries = currentEateries.filter(\.isOpen)
+        var openEateries = currentEateries.filter(\.isOpen)
+        
+        LocationManager.shared.$userLocation
+        .sink { userLocation in
+            openEateries = openEateries.sorted(by: { eatery1, eatery2 in
+                let dist1 = eatery1.walkTime(userLocation: userLocation)
+                let dist2 = eatery2.walkTime(userLocation: userLocation)
+                guard let dist1 else { return true }
+                guard let dist2 else { return true }
```

Comment:
New line should go after the guard. Also, I believe lines 216 and 217 can be condensed to:
`guard let dist1, let dist2 else { return true }`